recipes/xstatic-bootstrap/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "XStatic-Bootstrap" %}
+{% set version = "4.1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1800e6bb5fc687604d8a893eee8c7882d800a6f3d6721799016f99a47d1dac0f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - xstatic.pkg.bootstrap
+
+about:
+  home: https://getbootstrap.com/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Twitter Bootstrap packaged for setuptools'
+
+  description: |
+    Quickly design and customize responsive mobile-first sites with Bootstrap, the world’s most popular front-end open source toolkit, featuring Sass variables and mixins, responsive grid system, extensive prebuilt components, and powerful JavaScript plugins.
+  doc_url: https://getbootstrap.com
+  dev_url: https://github.com/twbs/bootstrap
```

I think this should probably be:
```suggestion
  dev_url: https://github.com/xstatic-py/xstatic-bootstrap
```

---

ah ok, that's a start, I asked Thomas if he has access to the recent one.